packages/headless/src/api/analytics/case-assist-analytics.ts

```diff
@@ -27,18 +27,12 @@ export type StateNeededByCaseAssistAnalytics = ConfigurationSection &
   Partial<CaseInputSection> &
   Partial<DocumentSuggestionSection>;
 
-export class CaseAssistAnalyticsProvider implements CaseAssistClientProvider {
-  private state: StateNeededByCaseAssistAnalytics;
-  constructor(getState: () => StateNeededByCaseAssistAnalytics) {
-    this.state = getState();
-  }
-
+export class CaseAssistAnalyticsProvider
+  extends BaseAnalyticsProvider<StateNeededByCaseAssistAnalytics>
+  implements CaseAssistClientProvider
+{
   public getSearchUID() {
-    return null as unknown as string;
-  }
-
-  public getOriginLevel1() {
```

Comment:
We don't have an originLevel1 now? Or was this not used?

Reply:
its declared in the base-analytics which is extended by the class